quic/s2n-quic/src/tests.rs

```diff
@@ -691,3 +691,85 @@ fn increasing_pto_count_under_loss() {
         pto_count
     );
 }
+
+// TODO: https://github.com/aws/s2n-quic/issues/1726
+//
+// The rustls tls provider is used on windows and has different
+// build options than s2n-tls. We should build the rustls provider with
+// mTLS enabled and remove the below `cfg_attr(ignore)`.
+#[cfg_attr(any(target_os = "windows"), ignore)]
```

Comment:
You need to completely disable compilation 
```suggestion
#[cfg(any(target_os = "windows"))]
```